content/en/blog/_posts/2022-11-11-retroactive-default-storage-class.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: blog
+title: "Retroactive default StorageClass"
+date: 2022-11-11
+slug: retroactive-default-storage-class
+---
+
+**Author:** Roman Bednář (Red Hat)
+
+In Kubernetes 1.25 a new Alpha feature was introduced to improve the way default StorageClass is assigned to a PersistentVolumeClaim (PVC). It is no longer required to create a default StorageClass first and PVC second in order to assign the class and any PVCs without StorageClass can be updated later. The feature will be Beta in Kubernetes 1.26.
```

Comment:
```suggestion
As an optional, alpha feature, the v1.25 release of Kubernetes introduced a change to the way
that a default StorageClass is assigned to a PersistentVolumeClaim (PVC). With that feature enabled,
you no longer need to create a default StorageClass first and PVC second in order to assign the
class; also any PVCs without a StorageClass assigned can be updated later.
The new behavior will become beta in Kubernetes 1.26.
```

Is this right? Also, is this on by default for v1.26?

Reply:
Yes, and it is on by default in 1.26: https://github.com/kubernetes/kubernetes/pull/113329